docker/Dockerfile

```diff
@@ -34,7 +34,7 @@ RUN --mount=type=cache,id=apt-dev,target=/var/cache/apt \
     python3.8-distutils \
     python3.8-venv \
     python3-venv \
-    openjdk-11-jre-headless \
+    openjdk-17-jdk \
```

Comment:
could you please add comments (eg. link of the jdk headless ca-certificate issue) at here so that we can follow later?

Reply:
updated.